backend/src/controllers/writeDetailsToFile.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs')
+
+async function saveDetails(req, res) {
+    try {
+        let data = req.body
+        let requiredData = Object.keys(data)
+            .map((key, value) => { return [key, data[key]].join('='); })
+            .join('\n');
+        fs.writeFileSync('data.txt', `Date: ${new Date().toLocaleString()} \n${requiredData}\n\n`, { flag: 'a+' }, err => {
+
+            if (err) {
+               return res.send("err", err)
```

Comment:
there should be some error message. cant be only "err"

Reply:
done